### ISSUE TITLE
AMFIPass.kext

### ISSUE THREAD

**Issue by kaoskinkae** [headkaze/Hackintool#214]
It is possible to add the kext AMFIPass.kext so that it can be updated in the list of extensions to be downloaded and would better be the kext needed for OCPL

**Comment by benbaker76** [comment]
Yes there's a way to do it by adding it to [kexts.plist](https://github.com/benbaker76/Hackintool/blob/master/Resources/Kexts/kexts.plist)

**Comment by kaoskinkae** [comment]
Thank you, let's see if it is translated into Spanish for the next version
